### PR TITLE
ADD: Spanish-resources

### PR DESCRIPTION
**Pull request by Deepanshu0810**
## Description
In this PR I have added resources in Spanish language section

## Checks
- [x] Proper Indexing
- [x] Correct Spellings 
- [x] Verified Resources

### CODE REVIEW

File: ES(Spanish)/README.md [hunk 1]
```diff
@@ -42,8 +42,36 @@ Para añadir o contribuir lee  [Contribution-guide](https://github.com/Ashish-kh
...
-  
+## Índice
+- [Docker](#docker)
+- [Lenguajes de programación](#lenguajes-de-programación)
+    * [Python](#python)
+- [Desarrollo front-end](#desarrollo-front-end)
+    * [HTML](#html)
+    * [JavaScript](#javascript)
+
+
```
Comment (Ashish-khanagwal) [inline ES(Spanish)/README.md:53]: please remove that extra line `53`

File: ES(Spanish)/README.md [hunk 2]
```diff
@@ -42,8 +42,36 @@ Para añadir o contribuir lee  [Contribution-guide](https://github.com/Ashish-kh
...
+---
+## Docker
```
Comment (Ashish-khanagwal) [inline ES(Spanish)/README.md:55]: insert an extra empty line between `54 and 55`, `58 and 59`, & `70 and 71`

File: ES(Spanish)/README.md [hunk 3]
```diff
@@ -42,8 +42,36 @@ Para añadir o contribuir lee  [Contribution-guide](https://github.com/Ashish-kh
...
   
-## indice  
-  
+## Índice
+- [Docker](#docker)
+- [Lenguajes de programación](#lenguajes-de-programación)
+    * [Python](#python)
+- [Desarrollo front-end](#desarrollo-front-end)
+    * [HTML](#html)
+    * [JavaScript](#javascript)
```
Comment (Ashish-khanagwal) [inline ES(Spanish)/README.md:51]: add CSS in the index as well
```
[CSS](#css)
```

File: ES(Spanish)/README.md [hunk 4]
```diff
@@ -42,8 +42,36 @@ Para añadir o contribuir lee  [Contribution-guide](https://github.com/Ashish-kh
   
 ---
   
-## indice  
-  
+## Índice
+- [Docker](#docker)
+- [Lenguajes de programación](#lenguajes-de-programación)
+    * [Python](#python)
+- [Desarrollo front-end](#desarrollo-front-end)
```
Comment (Ashish-khanagwal) [inline ES(Spanish)/README.md:49]: Add this `[Desarrollo front-end](#desarrollo-front-end)` in alphabetical order (above Docker) and its resources section too above Docker's resources.

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

**Comment by Deepanshu0810** [comment]
Sure, will resolve the problem within few hours

**Review by Deepanshu0810 — COMMENTED** [review]
requested changes has been made, please verify

**Review by Ashish-khanagwal — APPROVED** [review]
Looks good

thank you for contributing

**Pull request merged**